File: client_packages_dev/js_packages/src/CharacterManager/CharacterManagerEvents.ts
export enum CharacterManagerEvents {
  CREATE_CHARACTER_FROM_CEF = "CEF:CLIENT:CharacterManager:CreateCharacter",
  UPDATE_PARENTS_FROM_CEF = "CEF:CLIENT:CharacterManager:UpdateParents",
  CHANGE_GENDER_FROM_CEF = "CEF:CLIENT:CharacterManager:ChangeGender",
  UPDATE_FACE_FEATURES_FROM_CEF = "CEF:CLIENT:CharacterManager:UpdateFaceFeatures",
  EXECUTE_CHARACTER_CREATION = "CEF:CLIENT:CharacterManager:ExecuteCharacterCreation",
  CHARACTER_CREATED_SUBMIT_FROM_CEF = "CEF:CLIENT:CharacterManager:CharacterCreatedSubmit",

  CREATE_CHARACTER_INIT_TO_SERVER = "CLIENT:SERVER:CharacterManager:InitCharacterCreation",
  CREATE_CHARACTER_FINISH_TO_SERVER = "CLIENT:SERVER:CharacterManager:CharacterCreationFinish",

  CREATE_CHARACTER_START = "SERVER:CLIENT:CharacterManager:CharacterCreationStart",
  GENDER_CHANGED_FROM_SERVER = "SERVER:CLIENT:CharacterManager:GenderChanged",
  CHARACTER_CREATED_FROM_SERVER = "SERVER:CLIENT:CharacterManager:CharacterCreated",
  APPLY_CHARACTER_FROM_SERVER = "SERVER:CLIENT:CharacterManager:ApplyCharacter",
}

File: dotnet/resources/CharacterManager/CharacterManagerEvents.cs
namespace CharacterManager
{
    internal static class CharacterManagerEvents
    {
        #region Character Select

        public const string GetOwnCharacters = "CEF:SERVER:CharacterManager:GetOwnCharacters";
        
        public const string SelectCharacter = "CEF:SERVER:CharacterManager:SelectCharacter";

        #endregion

        #region Character Create

        public const string InitCharacterCreationFromClient = "CLIENT:SERVER:CharacterManager:InitCharacterCreation";

        public const string CharacterCreationStart = "SERVER:CLIENT:CharacterManager:CharacterCreationStart";

        public const string ChangeGenderFromCef = "CEF:SERVER:CharacterManager:ChangeGender";

        public const string CharacterCreatedSubmitFromClient = "CLIENT:SERVER:CharacterManager:CharacterCreationFinish";
        
        #endregion
    }
}